docs/Developer-guide/Building-GlusterFS.md

```diff
@@ -66,7 +67,7 @@ The following yum command installs the build requirements for CentOS / Enterpris
     pyliblzma python-devel python-eventlet python-netifaces           \
     python-paste-deploy python-simplejson python-sphinx python-webob  \
     pyxattr readline-devel rpm-build sqlite-devel systemtap-sdt-devel \
-    tar userspace-rcu-devel
+    tar userspace-rcu-devel liburing-devel
```

Comment:
For EL7, liburing rpm is not officially available. Centos-8 onwards has it though.

Reply:
Oh ok, I will remove it.

Reply:
So, should we add a note like:
```For CentOS 7, use:
./configure --disable-linux-io_uring
```

Reply:
I think we can add a note down below since this entire page is for source install.